src/TipBox.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="alert container" :class="[boxStyle, addClass, lightStyle]" :style="customStyle">
-        <div v-if="!isDefault" class="icon-wrapper">
+        <div v-if="!isDefault" class="icon-wrapper" :class="[iconStyle]" >
```

Comment:
Remove stray whitespace:

```suggestion
        <div v-if="!isDefault" class="icon-wrapper" :class="[iconStyle]">
```